cherry-picker/.gitignore

```diff
@@ -0,0 +1,2 @@
+lib
+node_modules
```

Comment:
Where does _lib_ come from? And isn't _node_modules_ already ignored?

Reply:
I guess _lib_ is fine to ignore. This package isn't compiled like _set-variables_. But if it were, it would be necessary to ignore _lib_.